src/components/home/Testimonial.astro

```diff
@@ -1,89 +1,55 @@
 ---
-const list = [
-  {
-    title: "Game-changer for Bitcoin Education!",
-    desc: "The questions and practical exercises. I think they are very well designed to make you deeply reflect on the studied topic. It help me a lot to consolidate knowledge. The kindle and super fast support from bitshala team and the rest of bitshala comunity. Sharing knowledge, experience and a lot of good stuff. I was awesome to me you all. I loved the day when instead of following the planned agenda and discussing the weekly questions, there was a debate on philosophical and profound topics about money and  bitcoin. It was great",
-    name: "Lazysatoshi, Learning Bitcoin via CLI",
-    image: "/testimonial/lazy.webp",
-  },
-  {
-    title: "Transformative Learning Community",
-    desc: "I feel elated to be a part of the first cohort of graduates from Bitshala, and I hope that it turns into a beacon of Bitcoin literacy in India. It's a tailor-made platform and an avenue of great resources  for those who are keen on traversing on a path to learning and imparting Bitcoin education.",
-    name: "Faisal, Learning Bitcoin via CLI",
-    image: "/testimonial/faisal.webp",
-  },
-  {
-    title: "Rich and Empowering Learning Experience",
-    desc: "The LBTCL-cohort was an unbelievably rich and empowering exercise in learning the finer details of a Bitcoin node and transactions. Raj and Setu have made the experience worth so much by their sheer enthusiasm and dedication, that it leaves a mark on you. The mentorship and the community at Bitshala is invaluable, and like Bitcoin, it will gradually then suddenly take the world by storm",
-    name: "Shreyan, Learning Bitcoin via CLI",
-    image: "/testimonial/shreyan.webp",
-  },
+import { testimonial } from "../common/list.js";
+import TestimonialMobile from "./TestimonialMobile";
+
+const imageList = [
+  ["/testimonial/1.webp", "/testimonial/2.webp"],
+  [
+    "/testimonial/3.webp",
+    "/testimonial/4.webp",
+    "/testimonial/5.webp",
+  ],
+  ["/testimonial/6.webp", "/testimonial/7.webp"],
```

Comment:
Pls make sure you remove all old images

Reply:
These are new images for the testimonial section